README.md

```diff
@@ -39,6 +39,11 @@ The `apm` command line has to be installed in your CI environment and available
 
 See the [Atom Package CI Scripts](https://github.com/atom/ci#atom-package-ci-scripts) documentation.
 
+_Note: If you are running multiple versions of Atom in CI (Stable + Beta),
```

Comment:
Maybe we can make the doc a bit more generic as some users might use build stages or other CI.

```md
**Note:** If you are running multiple versions of Atom in CI (Stable + Beta), ensure that the `semantic-release` command runs on the Stable channel (the build with the `ATOM_CHANNEL=stable` environment variable) as the Beta channel builds only provide the `apm-beta` CLI. If you are using [travis-deploy-once](https://github.com/semantic-release/travis-deploy-once) this can be achieve by setting the Stable channel build to be the last build to run or by using the [buildleaderid](https://github.com/semantic-release/travis-deploy-once#-b---buildleaderid) options.
```

Reply:
Good idea!

Used a bit different wording, and removed the reference to the environment variable as it was breaking the italization of the text.